CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Version number changes (major.minor.micro) in this package denote the following:
 - A minor version will increase if one or more packages contained in the Docker image add new, backwards-compatible features, or if a new package is added to the Docker image.
 - A major version will increase if there are any backwards-incompatible changes in any of the packages contained in this Docker image, or any other backwards-incompabile changes in the execution environment.
 
+## [2.8.0] - 2019-02-12
+
+- R -> 3.5.1
```

Comment:
More accurate to say "update rocker/verse to 3.5.1" here, right? I'd expect that more than the R version are being updated here.

Reply:
Good point